src/test/java/duke/tasks/TaskTodoTest.java

```diff
@@ -0,0 +1,36 @@
+package duke.tasks;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskTodoTest {
+    private static final String TEST_UNESCAPED_DESC = "VERTICAL | BAR";
```

Comment:
Really like the idea of DEFINE the string rather than writing it out every time. Neat and clean